plugins/AccountsService/AccountsService.cpp

```diff
@@ -219,21 +219,22 @@ AccountsService::PasswordDisplayHint AccountsService::passwordDisplayHint() cons
 
 QString AccountsService::pinCodePromptManager() const
 {
+    // TODO remove this line when setting backend is ready and uncomment all
+    return "SchemaPinPrompt.qml";
+    /*
     auto value = getProperty(IFACE_UBUNTU_SECURITY, PROP_PINCODE_PROMPT_MANAGER);
     if (!value.isValid()) {
```

Comment:
is that check usefull ?, dbus doesn't reply with default PinPrompt if not properly set ?